idelib/dataset.py

```diff
@@ -1243,7 +1279,11 @@ def units(self):
 
 
     def path(self):
-        return "%s, %s" % (self.parent.path(), self.session.sessionId)
+        """ Get the combined names of all the object's parents/grandparents.
+        """
+        if self.session.sessionId:
```

Comment:
should this do a `None`-check first, like `if self.session is not None and self.session.sessionId`? I see this same check in `updateTransforms`.

Reply:
@StokesMIDE still waiting to hear back on ^this

Reply:
Probably safe in this case, since the object will be instantiated through the parent, but there could be edge cases. Addressed in 0814c11c33689f485cfe13ae5cf765ee0b1db1a9.